project-icp/bee-pollinator-app
# Favicon and No Data

## Overview

When there are no yield values to report on, don't include the bar
chart which comes with its own "no data" text.  Instead, render our own
custom messages via app templates. The logic includes the ability to
include extra information only when yield is visible, including the
table and an explanation.

Also updates the favicon to ICP provided image.

Connects #210 
Connects #202 

## Demo
**Favicon**
![screenshot from 2017-05-05 15 45 08](https://cloud.githubusercontent.com/assets/1014341/25761731/dbda28ee-31a9-11e7-8816-cf00ca831768.png)

**Scenario no-data**
![screenshot from 2017-05-05 15 21 05](https://cloud.githubusercontent.com/assets/1014341/25761594/5717abd6-31a9-11e7-8358-a8cf8508bd2e.png)

**Scenario yes-data**
![screenshot from 2017-05-05 15 20 51](https://cloud.githubusercontent.com/assets/1014341/25761595/57183db2-31a9-11e7-9208-8b6112b5cc22.png)

**Compare yes and no-data**
![screenshot from 2017-05-05 15 20 39](https://cloud.githubusercontent.com/assets/1014341/25761596/5718ff4a-31a9-11e7-99d2-9c35ef6c8bc9.png)

## Testing
* Bundle, `scripty manage collectstatic` and refresh
* In the desert or an urban area where there are unlikely to be any crops, add a field
* Verify that the appropariate "No Data" messages are shown in Scenario tabs and Compare
* Verify that the chart, table and "relative yield" description are not show
* Add a crop to a scenario (or redo the area in a cropped area) and verify that the no data message are not shown and the data is.
* Verify the bee silhouette is the new favicon
